### ISSUE TITLE
marie is probably fuming lmao

### ISSUE THREAD

**Issue by rullinoiz** [machport/yaaa-no#1]
thanks for this tho

**Comment by CarlosTKZ-exe** [comment]
we’re you able to put it together?

**Comment by CarlosTKZ-exe** [comment]
> thanks for this tho

we’re you able to put it together

**Comment by rullinoiz** [comment]
> > thanks for this tho
> 
> we’re you able to put it together

no, its been purposely broken but its out now https://calamari.cc/ios

**Comment by coolknife666** [comment]
How do u add it?

**Comment by coolknife666** [comment]
> > > thanks for this tho
> 
> > 
> 
> > we’re you able to put it together
> 
> 
> 
> no, its been purposely broken but its out now https://calamari.cc/ios

How do u add it

**Comment by rullinoiz** [comment]
> > > > thanks for this tho
> > 
> > > 
> > 
> > > we’re you able to put it together
> > 
> > 
> > 
> > no, its been purposely broken but its out now https://calamari.cc/ios
> 
> How do u add it

you can't, ios version is down and they're rewriting the whole thing

**Comment by coolknife666** [comment]
> > > > > thanks for this tho
> > > 
> > > > 
> > > 
> > > > we’re you able to put it together
> > > 
> > > 
> > > 
> > > no, its been purposely broken but its out now https://calamari.cc/ios
> > 
> > How do u add it
> 
> you can't, ios version is down and they're rewriting the whole thing

When will it come back?

**Comment by rullinoiz** [comment]
> > > > > > thanks for this tho
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > we’re you able to put it together
> > > > 
> > > > 
> > > > no, its been purposely broken but its out now https://calamari.cc/ios
> > > 
> > > 
> > > How do u add it
> > 
> > 
> > you can't, ios version is down and they're rewriting the whole thing
> 
> When will it come back?

no idea, all i know is that it will come back eventually

**Comment by coolknife666** [comment]
> > > > > > > thanks for this tho
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > 
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > we’re you able to put it together
> 
> > > > > 
> 
> > > > > 
> 
> > > > > no, its been purposely broken but its out now https://calamari.cc/ios
> 
> > > > 
> 
> > > > 
> 
> > > > How do u add it
> 
> > > 
> 
> > > 
> 
> > > you can't, ios version is down and they're rewriting the whole thing
> 
> > 
> 
> > When will it come back?
> 
> 
> 
> no idea, all i know is that it will come back eventually
> 
> 
> 
> its also under a new name https://script-ware.com

Oh ok ty

**Comment by Quyen001** [comment]
> > > thanks for this tho
> > 
> > 
> > we’re you able to put it together
> 
> no, its been purposely broken but its out now https://calamari.cc/ios

web dead

**Comment by rullinoiz** [comment]
> > > > thanks for this tho
> 
> > > 
> 
> > > 
> 
> > > we’re you able to put it together
> 
> > 
> 
> > no, its been purposely broken but its out now https://calamari.cc/ios
> 
> 
> 
> web dead

read the thread, it's on https://script-ware.com now
